Wrong behavior when angle_start=angle_end

### Commit
0.1.6

### Steps to reproduce
roslaunch psen_scan_v2 psen_scan_v2.launch sensor_ip:=192.168.0.100 angle_start:=2 angle_end:=2


### Expected behavior
Program abortion with meaningful error message

### Observed behavior
[ERROR] [1618242581.417592336]: Received error "Unknown operation result code.". Shutting down now.

The scanner will send this message "Unknown operation result code." even with default angle range parameters until it is switch off and on again

--- a/standalone/include/psen_scan_v2_standalone/scan_range.h
+++ b/standalone/include/psen_scan_v2_standalone/scan_range.h
@@ -73,9 +73,9 @@ constexpr ScanRangeTemplated<min_angle, max_angle>::ScanRangeTemplated(const uti
     throw std::out_of_range("End angle out of range");
   }
 
-  if (start_angle > end_angle)
+  if (start_angle >= end_angle)
   {
-    throw std::invalid_argument("Start angle must be smaller or equal to end angle");
+    throw std::invalid_argument("Start angle must be smaller than end angle");
   }
 }
 

--- a/standalone/test/unit_tests/util/unittest_scan_range.cpp
+++ b/standalone/test/unit_tests/util/unittest_scan_range.cpp
@@ -64,6 +64,11 @@ TEST(ScanRangeTest, testEndAngleSmallerThanStartAngle)
   EXPECT_THROW(TestScanRange(util::TenthOfDegree(17), util::TenthOfDegree(15)), std::invalid_argument);
 }
 
+TEST(ScanRangeTest, testEndAngleEqualStartAngle)
+{
+  EXPECT_THROW(TestScanRange(util::TenthOfDegree(15), util::TenthOfDegree(15)), std::invalid_argument);
+}
+
 }  // namespace psen_scan_v2_standalone_test
 
 int main(int argc, char* argv[])

